plugins/woocommerce-admin/client/tasks/task.tsx

```diff
@@ -25,6 +25,19 @@ export const Task: React.FC< TaskProps > = ( { query, task } ) => {
 		optimisticallyCompleteTask,
 	} = useDispatch( ONBOARDING_STORE_NAME );
 
+	const updateBadge = useCallback( () => {
+		const badgeElement: HTMLElement | null = document.querySelector(
+			'.toplevel_page_woocommerce .remaining-tasks-badge'
+		);
+
+		if ( ! badgeElement ) {
+			return;
+		}
+
+		const currentBadgeCount = Number( badgeElement.innerText );
```

Comment:
Can we add a check here to prevent showing `0`?

It might be something like this:
```
if ( currentBadgeCount === 1 ) {
	badgeElement.remove();
}
```

Reply:
Good thought, done in https://github.com/woocommerce/woocommerce/pull/32611/commits/d3ff719428fe87649166064e351249131d84c74a